internal/pkg/template/templates/workloads/partials/cf/alb.yml

```diff
@@ -26,7 +26,7 @@ TargetGroup:
     {{- if .HTTPTargetContainer.IsHTTPS }}
     HealthCheckProtocol: HTTPS
     {{- end}}
-    Port: !Ref ContainerPort
+    Port: !Ref TargetPort
```

Comment:
Is the health check port already pointing to the target port? 

This `Port` field isn't useful 😂 https://stackoverflow.com/questions/42715647/whats-the-target-group-port-for-when-using-application-load-balancer-ec2-con/42823808#42823808 but it's good to be consistent

Reply:
You're right, it's not useful😂 But! This field does show up in the target group console so this should make it more obvious which port we are using.

Reply:
also to answer your first question - yes, health check port defaults to traffic port if not set manually.